### DISCUSSION TITLE
Broken terminal in Vscode

### DISCUSSION THREAD

**Discussion by smakevkin** [JanDeDobbeleer/oh-my-posh#5015] [Troubleshoot] [answered]
I can't understand why, in the VS Code terminal half broken, but everything is fine in powershell in windows itself, can you help?

VsCode terminal:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/50344064/971d580b-250b-454e-becf-971c7a52cb6a)

Windows Powershell:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/50344064/28c586d0-6943-4615-b888-2aea55daf693)

In Vscode font set to JetBrain Mono Nerd font
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/50344064/36860303-8e48-43a8-bd53-56325179ae06)

**Comment by smakevkin** [comment]
The main problem is that just yesterday everything worked fine, but today VSCode was updated, and oh-my-posh(June 7) was updated. Could it be related to this?

**Reply by JanDeDobbeleer** [reply, marked as answer]
It's a vscode bug. They are working on it. https://github.com/microsoft/vscode/issues/211922
